cms/static/js/views/course_info_update.js

```diff
@@ -8,8 +8,8 @@ define(["js/views/baseview", "codemirror", "js/models/course_update",
         // collection is CourseUpdateCollection
         events: {
             "click .new-update-button" : "onNew",
-            "click #course-update-view .save-button" : "onSave",
-            "click #course-update-view .cancel-button" : "onCancel",
+            "click .save-button" : "onSave",
```

Comment:
Note to reviewers: the element with ID course-update-view is outside the template used for this view.